tests/page-objects/login/login.po.js

```diff
@@ -20,6 +21,17 @@ const changeLocale = locale => {
   return element(by.css(`.locale[name="${locale}"]`)).click();
 };
 
+const getLabelForUser = () => {
```

Comment:
Any reason these can't be elements defined as const vs a function?

Reply:
i like constants in case like these, just went with existing code style. will make then constants